samples/cpp/benchmark_app/main.cpp

```diff
@@ -161,12 +190,16 @@ int main(int argc, char* argv[]) {
 
         // ----------------- 1. Parsing and validating input arguments
         // -------------------------------------------------
+        auto args_string = get_console_command(argc, argv);
```

Comment:
What is a reason for doing it inside tool?

Reply:
It is easier to verify which benchmark_app is being used, C++ or Python, and from which directory. It can happen for example when the PYTHONPATH/PATH env variable is modified, or after the output alignment PR is merged, when both benchmarks' output is indistinguishable.

Reply:
> It is easier to verify which benchmark_app is being used, C++ or Python, and from which directory. It can happen for example when the PYTHONPATH/PATH env variable is modified, or after the output alignment PR is merged, when both benchmarks' output is indistinguishable.

Validation related stuff should not be included into the tool. Why this PR was merged? cc @p-durandin

Reply:
As pointed out by @helena-intel in the ticket, the idea of this change is to make the usage of the tool and sharing output logs easier for the customers and our coworkers alike, I think this is a valid addition to the tool

Reply:
@PiotrKrzem I still don't understand why it is problem to share logs with initial command line. If user have ran tool, he knows what he typed in command line.

Reply:
Moreover for some automation scenarios we increase final log size.